sqlQueries/nlstCohort.sql

```diff
@@ -138,7 +146,8 @@ geometryChecks AS (
       -- add " idc_data/" which acts as destination folder for s5cmd and
       -- at end for each row separated by new line character "\n" so it will be
       -- ready for creating a manifest file later in the workflow
-      string_agg (CONCAT("cp ",REPLACE(sopInstanceUrl, "gs://", "s3://"), " idc_data/"), "\n") as s5cmdUrls
+      --string_agg (CONCAT("cp ",REPLACE(sopInstanceUrl, "gs://", "s3://"), " idc_data/"), "\n") as s5cmdUrls
+       ANY_VALUE(CONCAT("cp --show-progress s3",REGEXP_SUBSTR(sopInstanceUrl, "(://.*)/"),"/* idc_data/")) AS s5cmdUrls
```

Comment:
Is it a good idea to have `--show-progress` here? I thought it is better to not use it in the manifest, and instead (only if needed) add it to `s5cmd run`.

Reply:
I think it may be a personal choice. I always use --show-progress in the notebooks as it is the only way to know how many slices were downloaded and at what speed. I understand giving a choice of whether or not to use --show-progress in idc-index point of view, but here my preference in the notebooks is to have the URLs ready for the s5cmd run command. Unless you feel strongly about this, I'll keep it as it is for now.